### ISSUE TITLE
OSMStories?

### ISSUE THREAD

**Issue by harry-wood** [osmlab/basket#30]
Is this the repo powering http://osmstories.org ? If so, someone might label it more clearly as such here on github. Maybe even rename the repo? Also add a link from the website to here. 

...but maybe I'm misunderstanding

**Comment by d3netxer** [comment]
This is true, this repo is powering http://osmstories.org. I think re-naming the repo is a good idea, as well as a link to here from the website. I don't have permissions to edit the settings for this repo, but I can try to find someone who does.
